Automattic/wp-calypso
# Site Settings: Display global error/success notices when deleting a site

Fixes #158 by showing the following notice when users try to delete a site with active purchases.

<img width="766" alt="screen shot 2016-02-05 at 11 25 17" src="https://cloud.githubusercontent.com/assets/275961/12845172/1c480c12-cbfc-11e5-9050-ee3f5c8d73fd.png">

Pinging @mikeshelton1503 for a design review.

**Testing**
- Visit `/settings/delete-site/:site` for a site with active subscriptions.
- Attempt to delete the site.
- Assert that you see the error notice above.
- Visit `/settings/delete-site/:site` for a site without active subscriptions.
- Attempt to delete the site.
- Assert that you see a success message when the site is deleted.
